docs/reference/commandline/swarm_join_token.md

```diff
@@ -0,0 +1,52 @@
+<!--[metadata]>
++++
+title = "swarm join-token"
+description = "The swarm join-token command description and usage"
+keywords = ["swarm, join-token"]
+advisory = "rc"
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# swarm init
+
+```markdown
+Usage:	docker swarm join-token [--rotate] (worker|manager)
+
+Manage join tokens
+
+Options:
+      --help     Print usage
+      --rotate   Rotate join token
+```
+
+Join tokens are the strings passed to the `--token` flag of `swarm join`. This command allows the
+currently valid tokens to be accessed and rotated.
+
+Run with only a single `worker` or `manager` argument, it will print the valid token for that
+role:
+
+```bash
+$ docker swarm join-token worker
+SWMTKN-1-3pu6hszjas19xyp7ghgosyx9k8atbfcr8p2is99znpy26u2lkl-1awxwuwd3z9j1z3puu7rcgdbx
+$ docker swarm join-token manager
+SWMTKN-1-3pu6hszjas19xyp7ghgosyx9k8atbfcr8p2is99znpy26u2lkl-7p73s1dx5in4tatdymyhg9hu2
+```
+
+When used with the `--rotate` flag, a new join token will be generated for the specified role,
```

Comment:
Use the `--rotate` fag to generate a new join token for the specified role and print the output: